src/main/java/com/slimebot/games/wordchain/Wordchain.java

```diff
@@ -0,0 +1,179 @@
+package com.slimebot.games.wordchain;
+
+import com.slimebot.games.Game;
+import com.slimebot.main.Main;
+import com.slimebot.games.GamePlayer;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.Channel;
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.requests.RestAction;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
+
+public class Wordchain extends Game<WordchainPlayer> {
+    private String lastWord;
+    private List<String> words = new ArrayList<>();
+    private int playerTurn;
+    private int round;
+    private ScheduledFuture<?> scheduledFuture;
+    private final short seconds;
+    private final short lives;
+
+    public Wordchain(long gameMaster, long channel, long guildId, short seconds, short lives) {
+        super(gameMaster, guildId, true, id -> new WordchainPlayer(id, lives));
+        this.seconds = seconds;
+        this.lives = lives;
+
+        this.minPlayers = 2;
+        this.maxPlayers = 10; // ????
+
+        Main.jdaInstance.getChannelById(MessageChannel.class, channel).sendMessage(players.get(0).getAsMention() + " hat ein Neues Wortketten Minspiel erstellt!").queue(message -> {
+            sendJoinMessageInThread(message.createThreadChannel("Wordchain " + uuid));
+        });
+    }
+
+    @Override
+    public void start() {
+        super.start();
+
+        sendMessage("Das Game hat begonnen! (" + uuid + ")").queue(); // TODO: better message
+
+        round = 0;
+        playerTurn = 0;
+
+        this.lastWord = null;
+
+        nextTurn();
+    }
+
+    @Override
+    protected EmbedBuilder buildJoinEmbed() {
+        return new EmbedBuilder()
+                .setColor(Main.embedColor(String.valueOf(guildId)))
+                .setTitle("Neues Wortketten Spiel erstellt!")
+                .setDescription("Um zu erfahren wie \"Wortkette\" funktioniert nutze ```/wordchain explanation```") // TODO
+                .addField("Spielleiter:in:", "<@"+gameMaster+">", true)
+                .addField(":timer: Timeout:", seconds + "s", true)
+                .addField(":x: Max Fehler:", lives + " Fehler", true)
+                .addField("Spieler:", players.stream().map(p -> p.getAsMention()).collect(Collectors.joining("\r\n")), true)
```

Comment:
Create method in Game that returns a player list as string
Just use `\n` without `\r`